### PR TITLE
LPS-156767 Simplify logic using the new attribute

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/upgrade/v2_9_4/FragmentEntryLinkUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+/**
+ * @author Eudaldo Alonso
+ */
+public class FragmentEntryLinkUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		if (!hasColumnType("FragmentEntryLink", "type_", "INTEGER")) {
+			alterColumnType("FragmentEntryLink", "type_", "INTEGER");
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/upgrade/v2_9_4/FragmentEntryLinkUpgradeProcess.java:47]: I think here you should use `alterTableAddColumn` and also `hasColumn`, isn't it?

Comment (victorg1991) [reply modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/upgrade/v2_9_4/FragmentEntryLinkUpgradeProcess.java:47]: @ealonso

Comment (ealonso) [reply modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/upgrade/v2_9_4/FragmentEntryLinkUpgradeProcess.java:47]: 😂

Comment (ealonso) [reply modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/upgrade/v2_9_4/FragmentEntryLinkUpgradeProcess.java]: fixed

**Comment by lfbesada** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/8651

**Pull request closed without merging**